content/docs/replenishment/R and R forms/index.md

```diff
@@ -58,12 +58,6 @@ The first time you create an R&R form, you will need to select each of these opt
 
 ![Add R&R: pre-filled](images/add_rnr.png)
 
-The first time you create an R&R form, you can select any available period. After that, you will only be able to select the next period in the sequence.
```

Comment:
Might be worth noting that if you skip periods, it will not prepopulate with values from previous rnr forms?